rr_common/include/rr_common/RelativePoseHistoryClient.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include <ros/ros.h>
+#include <geometry_msgs/Pose2D.h>
+#include <nav_msgs/Path.h>
+
+
+namespace rr {
+
+class RelativePoseHistoryClient {
+public:
+  using Pose = geometry_msgs::Pose2D;
```

Comment:
This type renaming is of no use. Pull the Pose2D name from geometry_msgs, and do this in the .cpp file

Reply:
future improvement